src/stream/src/executor/dynamic_filter.rs

```diff
@@ -282,7 +282,7 @@ impl<S: StateStore> DynamicFilterExecutor<S> {
 
                     // The condition is `None` if it is always false by virtue of a NULL right
                     // input, so we save evaluating it on the datachunk
-                    let condition = dynamic_cond(right_val);
+                    let condition = dynamic_cond(right_val).transpose()?;
```

Comment:
`build_expr` error is impossible to recover. So it propagates error up rather than suppressing it like `eval_infallible`. Is this understanding correct? @xxchan 

(Also applies to hop window stream executor below.)

Reply:
yes